src/main/java/gregicadditions/recipes/chain/Dyes.java

```diff
@@ -524,18 +562,23 @@ public static void init(){
                 .outputs(CyanIndigoDye.getItemStack(2))
                 .buildAndRegister();
 
+        //C8H4O3 + 2 C6H6O2 -> 	C20H12O5 + 2H2O
         CHEMICAL_RECIPES.recipeBuilder().duration(240).EUt(1150)
                 .input(dust, PhthalicAnhydride, 15)
-                .fluidInputs(Resorcinol.getFluid(1000))
-                .outputs(Fluorescein.getItemStack())
+                .fluidInputs(Resorcinol.getFluid(2000))
+                .outputs(Fluorescein.getItemStack(37))
+                .fluidOutputs(Water.getFluid(2000))
                 .notConsumable(ZincChloride.getItemStack())
                 .buildAndRegister();
 
+        //C20H12O5 + 4 I + 2 NaOH -> C20H6I4Na2O5 + 4 H + 2 H2O
         CHEMICAL_RECIPES.recipeBuilder().duration(150).EUt(750)
                 .input(dust, Iodine, 4)
+                .input(dust, SodiumHydroxide, 6)
                 .inputs(Fluorescein.getItemStack(37))
-                .outputs(Erythrosine.getItemStack(37))
+                .outputs(Erythrosine.getItemStack(20))
```

Comment:
Not sure why this was changed, as it was good as 37
```suggestion
                .outputs(Erythrosine.getItemStack(37))
```